cluster/juju/layers/kubeapi-load-balancer/metadata.yaml

```diff
@@ -1,7 +1,10 @@
 name: kubeapi-load-balancer
 summary: Nginx Load Balancer
 maintainers:
-  - Charles Butler <charles.butler@canonical.com>
```

Comment:
@chuckbutler We can leave you on the maintainer lists if you want, we're mostly concerned about the non-existent email address. What would you prefer?